makefile

```diff
@@ -232,6 +232,8 @@ $(buildDir)/dist.tar.gz:$(buildDir)/make-tarball $(clientBinaries) $(uiFiles)
 	./$< --name $@ --prefix $(name) $(foreach item,$(distContents),--item $(item)) --exclude "public/node_modules" --exclude "clients/.cache"
```

Comment:
The makefile should have another variable for the location of the lobster working directory files (in `bin`) and should be deleted as part of `clean`.

Reply:
I tried that here https://github.com/evergreen-ci/evergreen/pull/4454/commits/5ba149ff55325746104fe12695c422ac11c6ff72#diff-beda42571c095172ab63437d050612a571d0d9ddd3ad4f2aecbce907a9b7e3d0R522